drupalux/seven
# Update navigation list styles

## Issue

https://github.com/drupalux/claro/issues/70

## Screenshot / UI changes

Compact mode is missing from the designs, please carefully check whether the suggested solution is fine or not.

<img width="669" alt="screen shot 2018-12-14 at 17 31 40" src="https://user-images.githubusercontent.com/9197058/50015179-2e49dc00-ffc6-11e8-9923-3ab875b4c591.png">

Figma lies about the color of the separator line. It says that it's the odl-silver (`#82828c`), but based on the Figma it's `#d8d9e0`. This color is not defined in our palette, so I just replaced it with the closest color from the palette: _light-gray_ `#d4d4d8`.

Screenshots here:

[Navigation screenshots - mobile and OSX.zip](https://github.com/drupalux/claro/files/2680983/Navigation.screenshots.-.mobile.and.OSX.zip)
[Navigation screenshots - Ubuntu and Windows.zip](https://github.com/drupalux/claro/files/2680984/Navigation.screenshots.-.Ubuntu.and.Windows.zip)

## Testing instructions

* Add the repository of the _Claro Distributon_`https://github.com/zolhorvath/seven-refresh/` to a Drupal project: `composer config repositories.'zolhorvath/clarodist' github
https://github.com/zolhorvath/clarodist.git`
* Add the `clarodist` package: `composer require zolhorvath/clarodist`
* Make sure that __this__ theme is installed to `[docroot]/themes/custom/` directory.
* Symlink `[docroot]/core/misc` to `[docroot]/themes/custom/misc`
* Run `yarn && yarn build:css` in the theme's root
* Install a Drupal with profile `clarodist`. 
* Enable the `navigation` module to get (access to) the test routes: `/admin`, `/admin/index` and `/navigation/config`
